redis-om-spring/src/main/java/com/redis/om/spring/repository/query/RedisEnhancedQuery.java

```diff
@@ -327,6 +392,14 @@ private Object executeQuery(Object[] parameters) {
       query.setSortBy(sortBy, sortAscending);
     }
 
+    if (hasLanguageParameter) {
+      Optional<SearchLanguage> maybeSearchLanguage = Arrays.stream(parameters).filter(SearchLanguage.class::isInstance).map(SearchLanguage.class::cast)
+          .findFirst();
```

Comment:
This line is too wide.
```suggestion
      Optional<SearchLanguage> maybeSearchLanguage = Arrays.stream(parameters).filter(SearchLanguage.class::isInstance)
          .map(SearchLanguage.class::cast).findFirst();
```